src/app/planos/componet/planos.component.ts

```diff
@@ -1,5 +1,8 @@
 import { Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
+import { Assinante } from 'src/app/assinante/service/assinante';
+import { AssinanteService } from 'src/app/assinante/service/assinante.service';
+import { RecommendationService } from 'src/app/homepage/service/recommendation.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8399665)
```suggestion
import { RecommendationService } from "src/app/homepage/service/recommendation.service";
```